homeassistant/components/fritzbox/config_flow.py

```diff
@@ -31,8 +31,8 @@
     }
 )
 
-RESULT_AUTH_FAILED = "auth_failed"
-RESULT_NOT_FOUND = "not_found"
+RESULT_AUTH_FAILED = "invalid_auth"
+RESULT_NOT_FOUND = "no_devices_found"
```

Comment:
Please adjust the constants as well to reflect the new key names.
```suggestion
RESULT_INVALID_AUTH = "invalid_auth"
RESULT_NO_DEVICES_FOUND = "no_devices_found"
```